python/ql/lib/change-notes/2022-10-04-api-subscript-nodes.md

```diff
@@ -0,0 +1,6 @@
+---
+category: minorAnalysis
+---
+* Fixed labels in the API graph pertaining to definitions of subscripts. Previously, these were labelled by `getMember` rather than `getASubscript`.
+* Added `API::SubscriptReadNode` and `API::SubscriptWriteNode` with convenience predicates to obtain the index and, in the case of a write, the written value.
+* Added convenience predicates to obtain a subscript at a specific index, when the index happens to be a statically known string.
```

Comment:
NIT: maybe we could mention what these `convenience predicates`, otherwise it'll be a bit of a treasure hunt for people that read this change-note without also viewing the diff :sweat_smile:

Reply:
Heh, yes!